src/Documentation/glossary.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
make is a JS file, so that we can pass through prettier

Reply:
@shcheklein The pre-commit hook takes care of that. We can just tweak the `npm run prettier-js` command to make it work for json files as well.
Something like this: `prettier --write '{pages,src}/**/*.{js,jsx,json}'`